docs/transport/braginskii.rst

```diff
@@ -0,0 +1,60 @@
+.. _braginskii:
+
+Braginskii
+**********
+
+The `~plasmapy.physics.transport.braginskii` subpackage includes a 
+class interface 
+`~plasmapy.physics.transport.braginskii.ClassicalTransport` with 
+multiple methods which are involved in modeling classical transport.
+
+.. topic:: Examples:
+
+   * :ref:`sphx_glr_auto_examples_plot_braginskii.py`
+
+.. _Classical transport models:
+
+Classical transport models
+==========================
+
+Broad overview of classical transport models implemented within 
+`braginskii.py`. These include:
+Braginskii
+Spitzer-Harm
+Epperlein-Haines (under construction)
+Ji-Held
+
+.. _Classical-transport-class:
+
+Classical transport class
+=========================
+
+Explain how to use the 
+`~plasmapy.physics.transport.braginskii.ClassicalTransport` interface.
+
+.. _Conductivities-resistivities:
+
+Conductivities and Resistivities
+================================
+
+Explain:
+resistivity
+thermoelectric_conductivity
+ion_thermal_conductivity
+electron_thermal_conductivity
+
+.. _Viscosities:
+
+Viscosities
+======================
+
+These include:
+ion_viscosity
+electron_viscosity
```

Comment:
Yep, these make sense, so these are the dynamic viscosities that you find for the classical plasma, similar to the viscosity of air or water or honey.  The big effect is the T^5/2 dependence, so as classical plasmas get hotter they become dramatically more viscous.  I think the ion viscosity typically dominates over the electron viscosity. The units are in Pa - s.  

For how this interacts with the magnetic field... apparently it makes everything a tensor. I'll have to get back to you on that.